tsuru/client/plugin.go

```diff
@@ -179,8 +180,17 @@ type PluginBundle struct {
 	fs  *gnuflag.FlagSet
 	url string
 }
+
+type BundleMetadata struct {
+	Name    string `json:"name,omitempty"`
+	Version string `json:"version,omitempty"`
+}
+
 type BundleManifest struct {
-	Plugins []Plugin
+	SchemaVersion  string            `json:"schemaVersion,omitempty"`
+	Metadata       BundleMetadata    `json:"metadata,omitempty"`
+	Plugins        []Plugin          `json:"plugins,omitempty"`
```

Comment:
The `BundleManifest` is intended for a completely different feature.
This should be two different types
```go
type BundleManifest struct {
	Plugins []Plugin
}

type PluginManifest struct {
	SchemaVersion  string            `json:"schemaVersion,omitempty"`
	Metadata       BundleMetadata    `json:"metadata,omitempty"`
	UrlPerPlatform map[string]string `json:"urlPerPlatform,omitempty"`
}
```

Reply:
another context note, the `PluginBundle` is intended for installing multiple plugins with a single url.
This PR is for installing a single plugin, with a single url (when the plugin could be one of many different binaries, declared in a `manifest.json`)